spec/models/claims/totals_spec.rb

```diff
@@ -140,65 +116,64 @@
   end
 
   context 'combined totals spec' do
+    subject(:claim) { create(:litigator_claim, :without_fees) }
+
     context 'LGFS' do
       context 'with VAT' do
-        it 'should add totals and calculate VAT as and when submodels are added or removed' do
-          claim = create :litigator_claim, :without_fees
-          allow(claim).to receive(:vat_registered?).and_return(true)
-          claim.apply_vat = true
-          expect(claim.vat_registered?).to be true
-          expect_totals_to_be(claim, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0)
+        before { allow(claim).to receive_messages(vat_registered?: true, apply_vat: true) }
 
+        it 'should add totals and calculate VAT as and when submodels are added or removed' do
+          is_expected.to have_totals({ fees_total: 0.0, fees_vat: 0.0, disbursements_total: 0.0, disbursements_vat: 0.0, expenses_total: 0.0, expenses_vat: 0.0 })
```

Comment:
This expectation is now much clearer, as we're able to see straight away which`field` is expected to have a given `value`. 🏆